File: src/main/java/baekjoon/N2292.java
package baekjoon;
import java.util.Scanner;

public class N2292 {

	public static void main(String[] args) {
		Scanner scan = new Scanner(System.in);
		long userInteger = scan.nextInt();
		scan.close();
		long lastNum = 1;
		long timeCal = 0;
		while (true) {
			if (lastNum <= userInteger && userInteger <= lastNum + timeCal * 6) {
				break;
			}
			lastNum += timeCal * 6;
			timeCal++;
		}
		System.out.println(timeCal+1);
	}

}

File: kotlin/src/main/java/com/soi/baekjoon/N2292.kt
package com.soi.baekjoon

import java.util.Scanner

class N2292 {

}

private fun getAnswer(userInteger:Long) :Long {
    var lastNumber = 1L;
    var timeCal = 0L;
    while (true) {
        if (lastNumber <= userInteger && userInteger <= lastNumber + timeCal * 6) {
            break
        }
        lastNumber += timeCal * 6
        timeCal++
    }
    return timeCal + 1;
}

fun main() {
    val scan = Scanner(System.`in`)
    println(getAnswer(scan.nextLong()))
}